Testing.ipynb

```diff
@@ -0,0 +1,1552 @@
+{
```

Comment:
If the notebook is part of the documentation then please rename it and move it to the docs.